### DISCUSSION TITLE
Multiple Contact Points Per Notification Policy

### DISCUSSION THREAD

**Discussion by valenvb** [grafana/grafana#37600] [Alerting]
It would be incredibly useful to be able add multiple contact points to a single notification policy. This would make contact points much more reusable so you don't have to have one for each possible combination of contact types.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
